docs/README.md

```diff
@@ -224,6 +224,14 @@ Here's an example configuration file for advanced settings and all of it's possi
         must_exclude:
           regex: 'DO NOT MERGE'
           message: 'Custom message...'
+          
+      files:
+		  # Check each modified file in the PR for the following starting at line 0.
+		  header: > 
+			   Entire license header.
+		  # only enforce it for these file patterns uses the same rules as CODEOWNER file
+		  pattern: *.go
+		  message: 'Custom message...'      
```

Comment:
Looks like some kind of tabbing issue? Can you fix?

Reply:
done